Makefile

```diff
@@ -11,14 +11,14 @@ VERSION ?= $(shell cat hokusai/VERSION)
 MINOR_VERSION ?= $(shell cat hokusai/VERSION | awk -F"." '{ print $$1"."$$2 }')
 
 dependencies:
-	pip install --requirement requirements.txt --quiet
-	pip install --quiet pyinstaller==3.3.1
+	pip install pipenv --quiet
```

Comment:
If we consider `pyenv` and `pipenv` prerequisites, I don't think we need to install `pipenv` here. Maybe add some installation documentation to the README?

```
$ brew install pyenv
$ pip install --user pipenv

Reply:
Hmmm... this is used for CI. I wouldn't expect someone to run this make file manually as a part of development.

Reply:
Or maybe that's what you mean... it's in the docker container, so maybe it isn't needed here..

Reply:
Yeah, if we're installing it in the docker image, I don't think we'd need this line here.

Reply:
Sounds good! :+1: